mozilla-release/browser/base/content/browser.js

```diff
@@ -7396,6 +7399,44 @@ var gRemoteTabsUI = {
   }
 };
 
+// Small utility handler which propagates current tab "privateness" property as
+// available to CSS selectors through #navigator-toolbox[on_private_tab]
+const _gPrivateTabUI = {
```

Comment:
uh, I know Firefox tends to have big files, but can we at least do not add our stuff into them? Is there some way of modularization?